packages/gatsby/src/utils/gatsby-webpack-eslint-graphql-schema-reload-plugin.ts

```diff
@@ -4,27 +4,33 @@
  *
  * This plugin replaces options of eslint-plugin-graphql during develop
  */
-const { store } = require(`../redux`)
+import { store } from "../redux"
 const eslintConfig = require(`./eslint-config`)
```

Comment:
we're fixing this in a different PR. I'm going to get that merged first so we can use an import statement on this one.

Reply:
Thanks for pointed out. Could you tell me the PR number?